samples/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
-if (HAS_QUOTE_PROVIDER)
+if (BUILD_ENCLAVES AND HAS_QUOTE_PROVIDER)
```

Comment:
Is there change in this PR that allows sample to be built when `BUILD_ENCLAVES` is `OFF`?

Reply:
Just modified `CMakeLists.txt` in the root folder so that `host_verify` sample can always be built regardless of the value of `BUILD_ENCLAVES`.